### PR TITLE
Fix error of readonly variable

### PR DESCRIPTION
**Pull request by anthonylgf**
Fixing the `./entrypoint.sh: line 13: tmpfile: readonly variable` error.

The executed steps were:
```shell
$ git@github.com:marcopaganini/check-trailing-spaces.git
$ cd check-trailing-spaces
$ ./entrypoint.sh
```

The system information:
- Ubuntu 20.04
- Bash: `5.0.17(1)-release (x86_64-pc-linux-gnu)`

### CODE REVIEW

**Comment by marcopaganini** [comment]
Thanks for the change but...
This is a bit of a catch-22 situation.

If we leave the readonly line, as it is now, it's clearly an error (and you're correct).
However, if we add readonly and shell expansion on the same line, shellcheck complains (it's a valid complain).
In this case, I think it's easier to remove the `readonly` keyword altogether.

**Comment by anthonylgf** [comment]
@marcopaganini I removed the `readonly` keyword too

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 *.swp
 *.bak
 *.*~
+changed_files.txt
```
Comment (marcopaganini) [inline .gitignore]: The last line on this file is missing a \n.

**Review by marcopaganini — APPROVED** [review]
No worries, I fixed the lack of CR at EOF for `.gitignore`.

**Pull request merged**